web-local/src/lib/application-state-stores/common-store.ts

```diff
@@ -0,0 +1,77 @@
+import type { V1ReconcileError } from "@rilldata/web-common/runtime-client";
+import { Readable, writable } from "svelte/store";
+
+export type CommonEntityData = {
+  errors: Array<V1ReconcileError>;
+};
+
+/**
+ * Store to save common data across entities.
+ * Currently only has errors
+ */
+export type CommonEntityState = {
```

Comment:
I think the "common entity" terminology here is a bit confusing. Ideally, we mirror terminology we have in the backend.

What about calling the filename something like either `file-store.ts` or `catalog-objects-store.ts` or `catalog-entries-store.ts`? And then mirroring that terminology throughout the type/variable/function names here in the file.

Reply:
`file-artifact-store` makes more sense. Just `file-store` is a little too broad and `catalog` is only for valid files.